queue/new/queue.php

```diff
@@ -47,15 +47,15 @@ class="dcf-w-100% dcf-input-text"
 		<div ng-if="appointments.length > 0" ng-cloak>
 			<!-- Swimlane headers -->
 			<div class="dcf-grid dcf-grid-fifths@md dcf-col-gap-2">
-				<div class="dcf-d-flex dcf-jc-center"><h5>Awaiting</h5></div>
-				<div class="dcf-d-flex dcf-jc-center"><h5>Checked-In</h5></div>
-				<div class="dcf-d-flex dcf-jc-center"><h5>Paperwork Complete</h5></div>
-				<div class="dcf-d-flex dcf-jc-center"><h5>Preparing</h5></div>
-				<div class="dcf-d-flex dcf-jc-center"><h5>Complete</h5></div>
+				<div class="dcf-d-flex dcf-jc-center"><h5>Awaiting ({{awaitingAppointments.length}})</h5></div>
```

Comment:
nice add

Reply:
Yah, Mac said he used the queue to determine how many people were in each state of the process (to determine if they could handle a walk-in or a second return for someone, etc.), and I figured this would make it easier than manually counting